src/test/java/uk/gov/hmcts/sptribs/common/notification/CaseStayedNotificationTest.java

```diff
@@ -39,10 +41,10 @@ class CaseStayedNotificationTest {
     @Test
     void shouldNotifySubjectOfApplicationReceivedWithEmail() {
```

Comment:
Think these test names need updating since they do not notify that the application has been received.